releng/platform-definitions/platform-definition-2021-12/platform-definition-2021-12.target

```diff
@@ -2,6 +2,7 @@
 <!--
    Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
    Copyright (c) 2022, Datadog, Inc. All rights reserved.
+   Copyright (c) 2022, SAP, SE. All rights reserved.
```

Comment:
I think you can only add this line only if you introduce a new file in the repository.

Reply:
thanks for the info, I'm new to the process